examples/asa/scripts/transfer/gold-contract-sc.js

```diff
@@ -3,7 +3,7 @@
  * This file demonstrates the example to transfer MicroAlgos
  * from contract account (contract approval mode) to another according to smart contract (ASC) logic
 */
-const { TransactionType, SignType } = require('@algorand-builder/runtime/build/types');
+const { types } = require('@algorand-builder/runtime');
```

Comment:
can we do: 
```
const { TransactionType, SignType  } = require('@algorand-builder/runtime').types;
..
```
OR
```
const runtime = require('@algorand-builder/runtime');
const { TransactionType, SignType  } = runtime.types;
..
```
Or we can keep this as well.

Reply:
I don't have a preference.

Reply:
ok, keeping same with shorter variable.